compile/WINDOWS_INSTRUCTIONS.md

```diff
@@ -0,0 +1,59 @@
+### Steps to compile IRBEM x64 on Windows
+1. Download and install MSYS2 https://www.msys2.org/
+2. Following the installation steps and update the packages:
+```
+pacman -Syu
+```
+After packages are updated, the terminal will be closed.
+3. Run `MSYS2 MSYS` from the start menu and update the rest of the packages
+```
+pacman -Su
+```
+4. Install base-devel package (https://packages.msys2.org/group/base-devel)
+```
+pacman -S base-devel
+```
+5. Install mingw-w64-x86_64-toolchain package (https://packages.msys2.org/group/mingw-w64-x86_64-toolchain)
+```
+pacman -S mingw-w64-x86_64-toolchain
+```
+Note that this is a large package but provides easiest installation process.
+6. Install git package as it required by the IRBEM make file (https://packages.msys2.org/base/git)
+```
+pacman -S git
+```
+7. Close this terminal and open `MSYS2 MinGW x64`.
+8. Navigate to the IRBEM folder. In the following example it is located at c:\libs\IRBEM\
+```
+cd /c
+cd libs/IRBEM
+```
+9. (*) If your **have not cloned IRBEM library** you can do it now using `MSYS2 MinGW x64` terminal:
+```
+cd /c/libs/
+git clone https://github.com/PRBEM/IRBEM.git
+```
+10. Compile the IRBEM library
+```
+make OS=win64 ENV=gfortran64 all
```

Comment:
This instruction set doesn't include setting the path, so the build is likely to fail due to gfortran, etc. not being found.

Reply:
No! The MSYS2 MinGW x64 terminal already have all necessary paths. The PATH setting is needed only to run compiled library in the Windows environment, e.g. in Matlab.

Reply:
I was following the instructions in `README.md` and then just read over these, but... The issue is that folks have already been working in the MSYS2 MSYS terminal (where the paths are _not_ set) and then are being asked to open the MSYS2 MinGW x64 terminal (where the paths are), and it's easy to read it as "open the MSYS terminal", type MSYS in the search bar and then just hit enter when MSYS2 MSYS pops up. I still don't like the fact that the `compile/WINDOWS_INSTRUCTIONS.md` follows different steps to `README.md`, especially since `README.md` says "See the `README.install` or `compile/WINDOWS_INSTRUCTIONS.md` files for more details." It's not "more details" if the required steps are just different.

First-time or casual users have a habit of giving up quickly, and every chance to get something wrong is a good chance the potential user will walk away and try something different. I'm going to mark this as resolved, but I really do suggest rationalizing the windows installation instructions.